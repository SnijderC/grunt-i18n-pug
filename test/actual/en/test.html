<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Test page</title>
  </head>
  <body>
    <h1>Localised versions of <code>test.pug</code></h1>
    <quote>Let's talk about 1 tasty apple(s) and 5 spoiled pear(s) in a way that makes more sense.</quote>
    <p>Locale is "en"</p>
    <p>Wat zou je ervan vinden als ik je 1 smakelijke appel gaf?</p>
    <p>Wat zou je ervan vinden als ik je 5 bedorven peren gaf?</p>
  </body>
</html>