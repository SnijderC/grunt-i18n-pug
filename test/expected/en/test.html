<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Test page</title>
  </head>
  <body>
    <h1>Localised versions of <code>test.pug</code>
    </h1>
    <p>Locale: "en"</p>
    <h2>Working with filters</h2>
    <p>Fruits are often tasty.</p>
    <p>Jim's pears usually tasty.</p>
    <p>Frank's apples are usually tasty too.</p>
    <p>Comparing Jim's apples and Frank's pears is not fair.</p>
    <p>1 pear is just one..</p>
    <p>2 pears is better than 1</p>
    <h2>Working with javascript functions</h2>
    <ul>
      <li>Number of apples: 1</li>
      <li>Number of pears: 5</li>
    </ul>
    <p>What would you say if I gave you 1 tasty apple?</p>
    <p>What would you say if I gave you 5 spoiled pears?</p>
  </body>
</html>
